### PR TITLE
setup the enterprise docs

### PR DESCRIPTION
**Pull request by HarshCasper**
## Introduction

This PR introduces a new **[LocalStack Enterprise](https://localstack-docs-preview-pr-1385.surge.sh/user-guide/localstack-enterprise/)** section under the User Guides.

## Changes

- Migrates SSO docs to the new section.
- Migrates CI Analytics docs to the new section.
- Migrates Enterprise support to the new section.
- Add docs on the K8 operator. 
- Add docs on the Enterprise image.

### CODE REVIEW

**Comment by bartsdev** [comment]
We should move all enterprise related content to one place ideally, there are more items that point to enterprise content:
https://docs.localstack.cloud/tags/enterprise-plan/
https://docs.localstack.cloud/getting-started/help-and-support/#enterprise-support
https://docs.localstack.cloud/getting-started/help-and-support/#signing-up-for-enterprise-support

**Review by bartsdev — APPROVED** [review]

File: content/en/user-guide/localstack-enterprise/k8s-operator/index.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+kind: LocalStack
+metadata:
+  name: env-0
+  namespace: default
+spec:
+  image: localstack/localstack-pro:3.5.0
+  debug: trace
+
+  authToken: "<my-auth-token>" # Set your LocalStack auth token here
+  autoLoadPods: ["<my-cloudpod>"] # Set your Cloud Pods to automatically load them here
```
Comment (whummer) [inline content/en/user-guide/localstack-enterprise/k8s-operator/index.md:36]: maybe we could mark this as optional
```suggestion
  autoLoadPods: ["<my-cloudpod>"] # Set your Cloud Pods to automatically load them here (optional)
```

File: content/en/user-guide/localstack-enterprise/k8s-operator/index.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+## Installation
+
+To install the K8s operator, run the following commands:
+
+{{< command >}}
+$ kubectl apply -f https://raw.githubusercontent.com/localstack/localstack-k8s-operator/v0.2.0/release/crds.yaml
+$ kubectl apply -f https://raw.githubusercontent.com/localstack/localstack-k8s-operator/v0.2.0/release/controller.yaml
+{{< / command >}}
+
+You can then deploy a LocalStack in the following manner:
```
Comment (whummer) [inline content/en/user-guide/localstack-enterprise/k8s-operator/index.md:23]: nit: Maybe we could be more specific here and say something like: 
> You can then deploy a LocalStack instance by storing the following file content as `localstack.yml` and applying it against the cluster via `kubectl apply -f localstack.yml`. 

I guess that might be clear for most readers, but not for all. 🤷

**Review by whummer — APPROVED** [review]
Kudos for tackling this @HarshCasper ! 🚀  Great to see a new, separate Enterprise section evolving. Over time we can also augment it with the additional enterprise-first features we're building, like, full Chaos Engineering support, US Gov Cloud support, etc. 👍

**Pull request merged**